.travis.yml

```diff
@@ -1,17 +0,0 @@
-sudo: required
-services:
-  - docker
-language: python
-python:
-  - "3.7"
-before_install:
-  - python3 -m pip install --upgrade pip setuptools wheel
-install:
-  - pip3 install git+https://github.com/nextstrain/cli
-  - nextstrain version
-  - nextstrain check-setup
-  - nextstrain update
-script:
-  - mkdir -p data/
-  - cp -v example_data/dengue* data/
-  - nextstrain build . auspice/dengue_all.json
```

Comment:
Should the GitHub Actions version also only request `dengue_all.json` instead of all serotypes?

Reply:
Probably. Although I feel like `example_data` is small enough (50 sequences each group) to test snakemake branching.

Reply:
Nod. As long as it's all reasonably quick for a CI run, then I'm fine with that. :-) Just wanted to make sure it was an intentional change not an oversight.